lib/Onnxifi/Base.cpp

```diff
@@ -146,26 +156,42 @@ void Graph::run(
     phs.push_back(var);
   }
 
+  auto ctx = std::make_unique<Context>();
```

Comment:
make this llvm::make_unique.

@bertmaher recently fixed bunch of places to make glow g++-4.8.5 compatible.